src/components/alert/alert.scss

```diff
@@ -117,7 +117,6 @@ ion-alert input {
 
   font-size: inherit;
   line-height: initial;
-  text-align: left;
```

Comment:
Was not aware, reverting that commit.
I also added `text-align: left` in `text-start` for that^